_topic_map.yml

```diff
@@ -211,6 +211,9 @@ Topics:
 - Name: Creating a project as another user
   File: creating-project-other-user
   Distros: openshift-enterprise,openshift-origin
+- Name: Configuring project creation
```

Comment:
Is there a reason that this is under the Authentication section? I see the section above it is project-related, but it seems auth-specific because of creating as a different user. This seems more like general project information that maybe should be elsewhere. wdyt?

Reply:
Moved to a new Applications -> Projects subdir w/ 2 other projects-related assemblies.